www/source/docs/index.html.slim

```diff
@@ -0,0 +1,121 @@
+---
+title: Habitat documentation
+description: All of the encyclopedic knowledge of the habitat project.
+---
+section.docs--hero
+  .row
+    .columns.medium-12
+      .docs--hero--content
+        h2.docs--hero--heading Cloud Native Application Automation
+
+section.docs--intro
+  .row
+    .columns.medium-5
+      p.docs--hero--text
+        | From containers to stateful services, Habitat provides a consistent way to build, deploy and manage your applications.
```

Comment:
Change to: The fastest path from code to cloud native

Reply:
fixed by new commit